<!DOCTYPE html>
<html lang="en-US">

<head>
<!-- https://www.ncbi.nlm.nih.gov/projects/msaviewer/ -->
<!-- This viewer will expire in 1-2 months. Update or embed our own -->
<script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/projects/msaviewer/js/multialign.js"></script></head>


  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-199638391-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    gtag('config', 'UA-199638391-1');
  </script>

<meta charset="UTF-8">

	<title>
        SARS-CoV-2 variants
    </title>


<!-- Begin Jekyll SEO tag v2.7.1 -->
<title>SARS-CoV-2 variants | Open-source tracking for emerging SARS-CoV-2 variants that pose a risk based on COVID-19 vaccine genetics.</title>
<meta name="generator" content="Jekyll v3.9.0" />
<meta property="og:title" content="SARS-CoV-2 variants" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Open-source tracking for emerging SARS-CoV-2 variants that pose a risk based on COVID-19 vaccine genetics." />
<meta property="og:description" content="Open-source tracking for emerging SARS-CoV-2 variants that pose a risk based on COVID-19 vaccine genetics." />
<link rel="canonical" href="http://localhost:4000/" />
<meta property="og:url" content="http://localhost:4000/" />
<meta property="og:site_name" content="SARS-CoV-2 variants" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="SARS-CoV-2 variants" />
<script type="application/ld+json">
{"description":"Open-source tracking for emerging SARS-CoV-2 variants that pose a risk based on COVID-19 vaccine genetics.","headline":"SARS-CoV-2 variants","url":"http://localhost:4000/","@type":"WebSite","name":"SARS-CoV-2 variants","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->

<link rel="preconnect" href="https://fonts.gstatic.com">
<link rel="preload" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" as="style" type="text/css" crossorigin>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="theme-color" content="#157878">
<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
<link rel="stylesheet" href="/assets/css/style.css?v=f0727b3a8f48d06eedacd54a03492ae0041d456b">
</head>

<body>
<!-- <a id="skip-to-content" href="#content">Skip to the content.</a> -->
<header class="page-header" role="banner">
<!-- <div class="logoimage"> -->
<!-- <img style="margin-bottom:-16px" -->
<!-- src="/images/logos/logo4.2_300_grey.png" -->
<!-- alt="Logo image" width="35.7" height="55" /> -->
<!-- </div> -->

<div class="project-name" >
	<h1> SARS-CoV-2 variants </h1>
  <h2 class="project-tagline">Open-source tracking for emerging SARS-CoV-2 variants that pose a risk based on COVID-19 vaccine genetics.</h2>
  <!-- 
    <a href="https://github.com/DylanLawless/SARS-CoV-2-VOC.github.io" class="btn">View on GitHub</a>
	 -->
  
</div>  
</header>

<!-- MSA viewer -->
<div id="msaviewer_x6fnt" class="MultiAlignViewerApp" data-autoload>
<a href="?embedded=true&anchor=-1&coloring=fbd&key=pRY_z7kUZs3KOijK6SseNE6aNkhpOWc8azpDLFcoRQbUCmFht1uDivkLzMQh61SWBY5Ymka-HbtaoU6sSKpEsXaYS5Znqk0,rB82xrAdb8TDMyHD4CIXPUeTP05gP246YjxKKl4uTADdDGhnvl3zKFwSaczBx3G6IKJ9tmOSOJd_jWuAbYZhnVO0brpChmg&track_config=protein_default&from=608&to=756&columns=d:120,b:55,x:17,aln,e:55,o:150&appname=ncbi_msav_demo"></a>
</div>

	<!-- <div id="7db9ZE" class="SeqViewerApp" data-autoload> -->
<!-- <a href="?embedded=true&url=https://raw.githubusercontent.com/DylanLawless/SARS-CoV-2-VOC.github.io/main/UCSCGenes_exon20.bed&appname=MyCompany_MyApp"></a> -->
<!-- </div> -->

    <main id="content" class="main-content" role="main">
      <p>If our live vaccine sequence alignment viewer has problems loading in your browser, <a href="https://www.ncbi.nlm.nih.gov/projects/msaviewer/?anchor=-1&amp;coloring=fbd&amp;key=pRY_z7kUZs3KOijK6SseNE6aNkhpOWc8azpDLFcoRQbUCmFht1uDivkLzMQh61SWBY5Ymka-HbtaoU6sSKpEsXaYS5Znqk0,rB82xrAdb8TDMyHD4CIXPUeTP05gP246YjxKKl4uTADdDGhnvl3zKFwSaczBx3G6IKJ9tmOSOJd_jWuAbYZhnVO0brpChmg&amp;track_config=protein_default&amp;from=608&amp;to=756&amp;columns=d:120,b:55,x:17,aln,e:55,o:150">you can visit it here via NCBI.</a></p>

<h1 id="variants-of-concern-aligned-to-sars-cov-2-vaccine-coding-sequences">Variants of concern aligned to SARS-CoV-2 vaccine coding sequences</h1>
<p>In the event of SARS-CoV-2 vaccine-resistant strain emergence, we would like to quickly determine which vaccines are affected.
Differences in national vaccination coverage, vaccine efficacy, viral phylogenetics, and infection reporting means that a robust statistical analysis of vaccine-resistance will be unlikely to return actionable findings as quickly as needed.
Understanding each vaccine design and having a detailed comparison of vaccine coding sequences is critical for pre-emptively assessing the risk.</p>

<p>Disturbingly, despite the expectation of scientific rigor, almost no genetic sequence data has been published from vaccine producers. 
Some designers have provided a description of genetic modifications relative to a known reference genome - precluding protection of intellectual property as an explanation. 
Here we compile data from vaccine designers and major health organisations to define the similarities and differences between each vaccine.
We target the coding sequences for vaccines on the World Health Organization (WHO) Prequalification (PQ) Units’ Emergency Use Listing (EUL) and compare them against variants-of-concern in circulating SARS-CoV-2 strains.</p>

<ul>
  <li><em>Currently, there are no variants of high concern</em>.</li>
</ul>

<h2 id="aims-and-results-overview">Aims and results overview</h2>
<p><a href="../data/variants_of_concern_to_vaccine.pdf">Open PDF visualisation - variants_of_concern_to_vaccine.pdf</a></p>

<blockquote>
  <p>Note: This work has not been peer reviewed. Every care has been taken by the authors to report accurate and sourced material, however caution is advised. This material is provided under live development to promote open-science.</p>
</blockquote>

<ul>
  <li>To produce standardized alignments of vaccine sequences.</li>
  <li>Determine which are at risk to emerging variants.</li>
</ul>

<p>From the data presented within:</p>

<ol>
  <li>The translated amino acid sequences for all vaccines were derived.</li>
  <li>These were aligned to the SARS-CoV-2 reference amino acid sequences of spike (S) glycoprotein.</li>
  <li>Known variants-of-concern were then annotated and visualised.</li>
</ol>

<p>Variants-of-concern for five SARS-CoV-2 strains 
(<a href="https://www.cdc.gov/coronavirus/2019-ncov/variants/variant-info.html?CDC_AA_refVal=https%3A%2F%2Fwww.cdc.gov%2Fcoronavirus%2F2019-ncov%2Fcases-updates%2Fvariant-surveillance%2Fvariant-info.html">CDC 4 Jun 2021</a>)
are illustrated against the 
translated amino acid sequences of the vaccines;</p>

<ul>
  <li>Moderna mRNA-1273</li>
  <li>Pfizer/BioNTech BNT-162b2</li>
  <li>Janssen/Johnson &amp; Johnson Ad26.COV2-S</li>
  <li>Novavax NVX-CoV2373</li>
  <li>Curevac CVnCoV</li>
  <li>Sputnik V</li>
  <li>AstraZeneca AZD1222</li>
  <li>AstraZeneca AZD2816</li>
  <li>BBIBP-CorV Sinopharm <a href="pages/vaccine_seq_recovered">(no published data)</a></li>
</ul>

<p>and reference protein/genome sequences;</p>

<ul>
  <li>QHD43416.1 [MN908947.3] and</li>
  <li>YP_009724390.1 [NC_045512.2].</li>
</ul>

<p>The variants-of-concern are shown here, illustrated on the protein structure;
<a href="https://www.rcsb.org/structure/6ZOX">6ZOX.pdb DOI: 10.2210/pdb6ZOX/pdb</a>
<em>Structure of Disulphide-stabilized SARS-CoV-2 Spike Protein Trimer (x2 disulphide-bond mutant, G413C, V987C, single Arg S1/S2 cleavage site),</em>
provided by Xiong et al., 2020 <a href="https://www.nature.com/articles/s41594-020-0478-5">10.1038/s41594-020-0478-5</a>.</p>

<ul>
  <li><a href="/pdb/renders/6zox_voc_20210604_front.jpg">6zox_voc_20210604_front.jpg</a></li>
  <li><a href="/pdb/renders/6zox_voc_20210604_side.jpg">6zox_voc_20210604_side.jpg</a></li>
  <li><a href="/pdb/renders/6zox_voc_20210604_top.jpg">6zox_voc_20210604_top.jpg</a></li>
</ul>

<div style="text-align:center"><img src="./pdb/renders/6zox_voc_20210604_stage.gif" width="80%" /></div>

<p>Two of the defining genetic features that are different between vaccines are seen here,</p>
<ul>
  <li>the S glycoprotein furin cleavage modification region (p.682-685)</li>
</ul>

<p><img src="/images/snapshot_furin_cleavage_region.png" alt="" /></p>

<ul>
  <li>the S glycoprotein stabilization modification region (p.986-987)</li>
</ul>

<p><img src="/images/snapshot_stabilizing_region_PP.png" alt="" /></p>

<p>Visual alignment is shown against translated coding sequence for spike glycoprotein,
illustrated here via</p>

<p><a href="https://nextstrain.org/sars-cov-2">nextstrain.org</a>.
<img src="/images/nextstrain_spike.png" alt="" /></p>

<p>Fasta sequences are included for:</p>

<ul>
  <li>Variants of Concern B.1.1.7</li>
  <li>Variants of Concern B.1.351</li>
  <li>Variants of Concern B.1.427</li>
  <li>Variants of Concern B.1.429</li>
  <li>Variants of Concern P.1</li>
  <li>Ref QHD43416.1 [MN908947.3]</li>
  <li>Ref YP_009724390.1 [NC_045512.2]</li>
  <li>mRNA-1273 vaccine translated</li>
  <li>BNT-162b2 vaccine translated</li>
  <li>Ad26.COV2-S vaccine translated</li>
  <li>NVX-CoV2373 vaccine translated</li>
  <li>Sputnik V alleged unmodified YP_009724390.1</li>
  <li>AZD1222 alleged unmodified YP_009724390.1</li>
  <li>AZD2816 modified YP_009724390.1 with variants from B.1.351</li>
</ul>

<h2 id="how-we-source-and-anlyze-our-data">How we source and anlyze our data</h2>

<p><a href="pages/Vaccine_list">Vaccine list</a> - Tables of all vaccines in trials and all vaccines with WHO EUL/PQ finalized reports that we use for our main analysis.</p>

<p><a href="/pages/Vaccine_sequences_summarised.html">Vaccine sequences summarised</a> - A short overview of the delivery methods and genetic modifications for vaccine coding sequences are required for our main analysis.</p>

<p><a href="/pages/Vaccine_details.html">Vaccine details</a> - A longer set of pertinent details about each vaccine that are required for interpreting the risk upon emergence of vaccine resistant variants.</p>

<p><a href="/pages/Naming_SARS-CoV-2_variants.html">Naming SARS-CoV-2 variants</a> - Technical details of the established nomenclature systems for naming and tracking SARS-CoV-2 genetic lineages. Additional recommendations for tracking variants-of-concern based on coding position coordinates and aligning all WHO EUL/PQ finalized vaccine coding sequences for pre-emptive planning.</p>

<p><a href="/pages/Vaccine_genetic_design.html">Vaccine genetic design</a> - Overview of the coding sequence genetics used by vaccine designers. Illustrates the similarities and differences between each vaccine, with or without genetic modification for an improved immune response. Critical for establishing the effect of variants-of-concern that may affect multiple vaccination strategies.</p>

<p><a href="/pages/Vaccine_sequence_reproduction.html">Vaccine sequence reproduction</a> - Detailed information about the construction of our coding sequence alignments. Reproduction of vaccine genetic designs using primary literature and reference sequence data.</p>

<p><a href="/pages/Variants-of-concern.html">Variants-of-concern</a> - Overview of the variants-of-concern as reported by the CDC. Currently no variants of high-concern are reported.</p>

<p><a href="/pages/Variants-of-concern_aligned_to_vaccine.html">Variants-of-concern aligned to vaccine</a> - Details about the combination of variants-of-concern and vaccine coding sequences. Produces a visual map of SARS-CoV-2 variants against each vaccine coding sequence, which is responsible for S protein expression. Emergence of viral strains containing new S protein variants may produce epitopes that fail to initiate sufficient vaccine-stimulated immune recognition.</p>

<p><a href="/pages/Main_files.html">Main file for download</a> - The main files that might be useful to researchers, containing raw and processed data, are listed here together. Other files that are not listed but are present in the data folder will contain intermediate data.</p>

<p><a href="/notes.html">Other notes</a> - Other notes with snippets and temporary, related material.</p>

<!-- ![](https://static01.nyt.com/newsgraphics/2020/04/02/virus-genome/assets/images/ribosome-600.png) -->
<div style="text-align:center"><img src="https://static01.nyt.com/newsgraphics/2020/04/02/virus-genome/assets/images/ribosome-600.png" /></div>

<h2 id="what-is-this">What is this?</h2>
<p>From humble research work to commercially-backed global vaccination, the scale of work in controlling COVID-19 due to SARS-CoV-2 infection has been vital and admirable. 
However, publication credibility is dependent on accurate scientific reporting. 
Vaccine producers have, in most cases, provided a description of their vaccine genetic sequence design in the primary literature. 
Release of coding sequence data is therefore not a matter of intellectual property protection.
It is concerning that all primary publications do not include the full coding sequence used.
Providing instead, only a description of variants introduces a hurdle to interpretation and detracts from scientific credibility. 
Some vaccine producers have never reported even the reference sequence, from which their unmodified sequence is derived. 
Naturally, each vaccine has a different efficacy; a variable level of protection from infection and immunological response.
Discussion of viral variant strains has become popular in the media. 
Should a vaccine-resistant strain develop, the resistance will be defined by the vaccine coding sequence.
We should not confound the facts about efficacy, variant strain, and vaccine genetic design.
The scientific community has a responsibility not only to measure post-vaccination susceptibility to new viral strains, but to determine <em>why</em> vaccine resistance may occur in reference to the vaccine coding sequence.
Currently, all vaccine genetic sequences have been almost identical (the main differences being modifications to the stabilization and furin cleavage regions). 
We want to be able to compare the genetic design of each vaccine; a requirement that should have been imposed during publication.
We have taken the responsibility of providing this data publicly - 
to answer our own call <em>“genetics on the table, please”.</em></p>
<blockquote>
  <p>“I am too familiar with the manner in which actual data are met with the suggestion that <em>other data, if they were collected, might show something else</em> to believe it to have any value as an argument. “Statistics on the table, please,” can be my sole reply. —Karl Pearson, 1910” Excerpt From: Stigler S.M. <a href="https://www.bookdepository.com/Statistics-on-the-Table/9780674009790">Statistics on the table the history of statistical concepts and methods.</a></p>
</blockquote>

<h2 id="about">About</h2>
<!-- ![](/images/location.png) -->
<div style="text-align:left"><img src="./images/location.png" width="50%" /></div>

<p>Location: <a href="https://goo.gl/maps/Xv1bY6Tgn4a5ccwN9">Switzerland, Canton Vaud</a></p>

<p>Created in support of open-science by: <a href="https://lawlessgenomics.com">https://lawlessgenomics.com</a></p>

<p><a href="https://dylan-lawless.github.io">Contact - Resume - CV</a></p>

<p>Host-pathogen genomic scientist. PhD (Medicine in Genetics and Immunology), MSc (Immunology), BSc (Microbiology)</p>

<p>Contribute via the original repository at <a href="https://github.com/DylanLawless/SARS-CoV-2-VOC.github.io">github.com/DylanLawless</a>.
With minimalism and privacy in mind, this site is built to be light and fast. 
This site containing no ads, cookies, &amp;c. However, one analytics tracker is included for counting page views to <a href="https://themarkup.org/blacklight?url=sarscov2variants.com">determine our resource requirements</a>.</p>

<p><a href="https://creativecommons.org/licenses/by/4.0/">CC-BY-4.0</a>.
<!-- https://www.paypal.com/donate?hosted_button_id=7K3CD4KDMKPBS --></p>


	<footer class="site-footer">
		
		<span class="site-footer-owner">
		<a href="https://lawlessgenomics.com"><img src="/images/logos/logo4.2_150ppi.png" 
		alt="Logo image" width="17.85" height="27.25" /></a>
		<a href="https://github.com/DylanLawless/SARS-CoV-2-VOC.github.io">SARS-CoV-2-VOC.github.io</a> is maintained by <a href="https://github.com/DylanLawless">DylanLawless</a>
		</span>
		
	<!-- <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a>.</span> -->
	<form action="https://www.paypal.com/donate" method="post" target="_top">
<input type="hidden" name="hosted_button_id" value="UU3F4ND2JAP2W" />
<input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donate_SM.gif" border="0" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate with PayPal button" />
<img alt="" border="0" src="https://www.paypal.com/en_IE/i/scr/pixel.gif" width="1" height="1" />
</form>
	<!--- https://www.paypal.com/donate?hosted_button_id=UU3F4ND2JAP2W --->
	</footer>
    </main>
  </body>
</html>
